src/components/Wizard/wizard.scss

```diff
@@ -86,14 +85,33 @@
 /* NewEditImportPane specific components */
 
 .new-form .form-buttons,
-.import-form .form-buttons {
+.import-form .form-buttons,
+.edit-form .form-buttons {
     margin-top: 1rem;
 
     .btn {
         @extend .col-2;
     }
 }
 
+.edit-form {
+    .edit-entry {
+        display: flex;
+    }
+
+    .form-check {
+        align-self: center;
+        width: 10%;
+    }
+
+    .btn-del {
+        border-radius: 3px;
+        font-size: 12px;
+        margin: 2px auto;
+        padding: 5px 10px;
+    }
```

Comment:
Prefer `em` and `rem` for consistent sizing. 0.75 em will be equivalent to 12px with the current body text size.

```
border-radius: 0.25em;
font-size: 0.75em;
margin: 0.25em 0;
padding: 0.5em 1em;
```

Also, add width: 100% so that btn will take up its full container and add a new style here:

```
.del-col {
   @extend .col-1;
}
```